test/e2e/population.dashboard.test.js

```diff
@@ -23,3 +24,11 @@ test.before(async t => {
     .expect(await populationDashboardPage.nodes.focusDay.exists)
     .notOk()
 })
+
+test('Population dashboard links', async t => {
```

Comment:
Very simple test to check that clicking into and out of the weekly page still works.